docs/jwst/associations/asn_from_list.rst

```diff
@@ -4,9 +4,8 @@ asn_from_list
 =============
 
 Create an association using either the command line tool
-``asn_from_list`` or through the Python API using either
-:class:`jwst.associations.asn_from_list.Main` or
```

Comment:
`jwst.associations.asn_from_list.Main` is not accessible. I could make it accessible but that would conflict with #9393 and change API behavior, so less controversial to just update the doc.